spec.html

```diff
@@ -8871,17 +8871,19 @@ <h1>[[Construct]] ( _argumentsList_, _newTarget_ )</h1>
     </emu-clause>
 
     <emu-clause id="sec-createbuiltinfunction" aoid="CreateBuiltinFunction">
-      <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _prototype_ ] ] )</h1>
-      <p>The abstract operation CreateBuiltinFunction takes arguments _steps_ and _internalSlotsList_ (a List of names of internal slots) and optional arguments _realm_ and _prototype_. _internalSlotsList_ contains the names of additional internal slots that must be defined as part of the object. This operation creates a built-in function object. It performs the following steps when called:</p>
+      <h1>CreateBuiltinFunction ( _steps_ [ , _additionalInternalSlotsList_ [ , _realm_ [ , _prototype_ ] ] ] )</h1>
+      <p>The abstract operation CreateBuiltinFunction takes arguments _steps_ and optional arguments _additionalInternalSlotsList_ (a List of internal slot names), _realm_, and _prototype_. _additionalInternalSlotsList_ contains the names of additional internal slots that must be defined as part of the object. This operation creates a built-in function object. It performs the following steps when called:</p>
       <emu-alg>
         1. Assert: _steps_ is either a set of algorithm steps or other definition of a function's behaviour provided in this specification.
         1. If _realm_ is not present, set _realm_ to the current Realm Record.
         1. Assert: _realm_ is a Realm Record.
         1. If _prototype_ is not present, set _prototype_ to _realm_.[[Intrinsics]].[[%Function.prototype%]].
+        1. Let _internalSlotsList_ be &laquo; [[Prototype]], [[Extensible]], [[Realm]], [[ScriptOrModule]] &raquo;.
```

Comment:
I'm not sure this is needed.

https://tc39.es/ecma262/#sec-ecmascript-function-objects already dictates that `[[Realm]]` and `[[ScriptOrModule]]` implicitly exist on a built-in function object, and https://tc39.es/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots dictates `[[Prototype]]` and `[[Extensible]]`, so they don't need to be explicitly specified.